README.md

```diff
@@ -1,6 +1,15 @@
 ## TIP Open WiFi Testing
 This repository contains the test automation framework and scripts for  TIP Open WiFi.
 
+## Cloud Controller build
+You can provision additional cloud controllers for your tests using [update cloud controllers](https://github.com/Telecominfraproject/wlan-testing/actions?query=workflow%3A%22update+cloud+controllers+build%22) build. To add additional cloud controller add another json object to the `testbeds` variable at line 19, where
+
+1. number == NOLA testbed number this cloud controller is assigned to
+2. version == docker images version to use with this particular deployment. supports 3 options:
+  1. "latest" - will use `0.0.1-SNAPSHOT` images
+  2. "" - will use `0.0.1-SNAPSHOT-yyyy-mm-dd` images from yesterdays date
+  3. "yyyy-mm-dd" - will use `0.0.1-SNAPHOST-yyyy-mm-dd` with the date provided (if they are available, if not build will fail).
+ 
```

Comment:
@4c74356b41 two questions:

1. Should we commit these changes to master to the whole flow to work? Can I branch out, make changes, run it manually to refresh the cloudsdk docker images, and then delete the branch to make the nightly pipeline take over again?
2. Are these changes picked up by the nightly... meaning - do we already refresh the cloudsdk docker images to yesterday's date every night?

Reply:
1. no, these are the readme changes?
2. depends on whatever is in the `testbeds`, if a particular instance is set to `""` - then yes, it will get refreshed once a new day starts for the agent (not sure whats the agent tz)

Reply:
General comment for this feature:
- would the release name change when to go Rel 1.0?
- changing the one should not affect other instances.

Reply:
1. define change? things in git do not change, unless somebody pushes changes
2. not sure I follow. which one (the image or the controller)? these settings are per controller, you cant be more granular than that

Reply:
@jaspreetsachdev to your questions:
1. It depends how we are going to tag the Docker images that are created after Rel 1.0. Assuming we will still use the `latest` tag - things here will continue to work without any additional changes. If we'll decide to use another Docker tag, lets say `rel-1.0-latest`, then we'll need to update the json here and commit to master. I wouldn't worry about that.
2. The tags are specified per each testbed. If you changes something in NOLA-1, this change will affect only the NOLA-1-related Cloud controller.